hyperspy/tests/component/test_voigt.py

```diff
@@ -39,20 +39,30 @@ def test_function():
     np.testing.assert_allclose(g.function(1), 2.97832092)
 
 
+def test_function_legacy():
+    g = Voigt(legacy=True)
+    g.area.value = 5
+    g.sigma.value = 0.5
+    g.gamma.value = 0.2
+    g.centre.value = 1
+    np.testing.assert_allclose(g.function(0), 0.78853024)
+    np.testing.assert_allclose(g.function(1), 2.97832092)
```

Comment:
Now that we know that the behaviour is correct, I think that we should remove this test as we are testing for future regression.